tools/gen_struct_info.py

```diff
@@ -231,15 +232,13 @@ def inspect_headers(headers, cpp_opts):
 
   js_file = tempfile.mkstemp('.js')
 
+  shared.check_sanity()
+  system_libs.Library.get_usable_variations()['libcompiler_rt'].get_path()
```

Comment:
`get_path`'s result is unused?

Reply:
Yes calling `get_path` just ensure that it exists.    Normally this would happen during library calculation but `-nostdlib` skips all of that.

Reply:
How about adding another function `.ensure()` that is identical to `get_path()` in functionality? Otherwise a comment here could help.

Reply:
Updated to actually use the result instead.